tests/functional/cylc-combination-scripts/07-vr-remote.t

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+# THIS FILE IS PART OF THE CYLC WORKFLOW ENGINE.
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+#------------------------------------------------------------------------------
+# Test `cylc vr` (Validate Reinstall restart)
+# Test that args for re-invocation are correct:
+export REQUIRE_PLATFORM='loc:remote runner:background fs:shared'
+. "$(dirname "$0")/test_header"
+
+set_test_number 4
+
+create_test_global_config '' """
+[scheduler]
+    [[run hosts]]
+        available = ${CYLC_TEST_HOST}
+
+"""
+
+# Setup
+WORKFLOW_NAME="cylctb-x$(< /dev/urandom tr -dc _A-Z-a-z-0-9 | head -c6)"
+cp "${TEST_SOURCE_DIR}/vr_workflow/flow.cylc" .
+sed -i 's@pause@stop --now --now@' flow.cylc
```

Comment:
Why do this `sed`?

Reply:
This error only turns up if the workflow is restarted by vr, not if it's reloaded, so we need the workflow in a stopped state.

Reply:
I think just `cat`ing to `flow.cylc` would be easier to follow than copying & modifying one from another test

Reply:
It's a single test workflow for all the VR's. I see your point, but having written a bunch of tests like this...

Although I swing around on this issue like a pendulum, I think that importing a workflow saved as a file rather than a heredoc is useful to analyzing broken tests - it's one less step to playing with the test workflow outside the test framework.

Reply:
Then I suggest making a `flow.cylc` for this test. This `sed` could easily break if the `vr_workflow/flow.cylc` is modified in future.